### ISSUE TITLE
From the homepage, click “View details” to navigate to the (new) program overview page

### ISSUE THREAD

**Issue by ssandbekkhaug** [civiform/civiform#8663]
# Description

Implement this behavior:
1. Open home page
2. Find a program card
3. Click "View Details"
4. (Expectation) navigate to program overview page

**Comment by rockycodes** [comment]
@ssandbekkhaug I'm about to start work on an epic to rethink how we're using that program overview page in the North Star: https://docs.google.com/document/d/1yiqHg3obWYrkbOmmcQptobJ97ky6uI3ttVukz-OfRME/edit

I'm pretty sure the work I'm doing is a blocker for launch in which case we don't need to re-implement the old behavior. But tagging @sijiayam to be sure.

**Comment by ssandbekkhaug** [comment]
That's right, this issue should represent "navigate to the new program overview page". Currently the button navigates to a custom link. I think we're on the same page :)

Updated the title

**Comment by melanie-exygy** [comment]
@ssandbekkhaug @rockycodes The current behavior when a user clicks "Program details" is to send the user to an external URL if that has been specified by the CF admin.  Only if the URL is not specified does it link to the program details view.  I don't think there's been any talk of removing the behavior of redirecting to an external link if there is one.  My understanding is that, Rocky, you'll essentially be replacing what happens in case #2, when there isn't an external link.  I suggest that we update this issue description to include the possibility of an external link, unless either of you has a different understanding.

**Comment by ssandbekkhaug** [comment]
Oh, I thought we were going to remove the custom external URL. @sijiayam What is the desired behavior?

**Comment by rockycodes** [comment]
@melanie-exygy I can confirm we're changing the behavior. The button will now always lead to the program preview page. Admin will be able to add an external url on that page if they would like to link out to more information.

@ssandbekkhaug we are on the same page! I'll likely make many mini issues to complete this work, but will use this as the overarching issue. I've started an epic board for this work and will be tagging related issues with the epic `[2024] Provide residents and TIs the information they need to complete a form at the start of the process`
